Employee_class.rb

```diff
@@ -14,4 +14,5 @@ def to_s
 end
 
  e1 = Employee.new("Siri", 5634234, "Female")
- puts e1.to_s
\ No newline at end of file
+ puts e1.to_s
+ 
```

Comment:
Can you make it more dynamic. After creating object of Employee, user need to insert name,id,gender,sal.